agent/consul/client.go

```diff
@@ -306,12 +316,6 @@ TRY:
 		return rpcErr
 	}
 
-	c.logger.Warn("Retrying RPC to server",
-		"method", method,
-		"server", server.Addr,
-		"error", rpcErr,
-	)
-
```

Comment:
Sorry for the confusion around this. We should put this back as this log would happen client side while the rate limiting log happen server side.